features/ContentCreation.feature

```diff
@@ -0,0 +1,235 @@
+    And Content attributes equals
+      | attribute | value                      |
+      | Title     | Test Article edited4       |
+      | Intro     | Test article intro edited4 |
+
+@javascript @common
+Scenario: Content moving can be cancelled
+  Given I navigate to content "Test Article edited4" in "Home"
+  When I click on the edit action bar button "Move"
+    And I open UDW and go to "Media/Images"
+    And I close the UDW window
+  Then I should be on content page "Test Article edited4" in "Home"
+    And Content attributes equals
+      | attribute | value                      |
+      | Title     | Test Article edited4       |
+      | Intro     | Test article intro edited4 |
+    And breadcrumb shows "Home/Test Article edited4" path
+
+@javascript @common
+Scenario: Content can be moved
+  Given I navigate to content "Test Article edited4" in "Home"
+  When I click on the edit action bar button "Move"
+    And I open UDW and go to "Media/Images"
+    And I confirm the selection in UDW
+  Then I should be on content page "Test Article edited4" in "Media/Images"
+    And Content attributes equals
+      | attribute | value                      |
+      | Title     | Test Article edited4       |
+      | Intro     | Test article intro edited4 |
+    And breadcrumb shows "Media/Images/Test Article edited4" path
+    And going to "Home", there's no "Test Article edited4" "Article" on "Sub-items" list
+
+@javascript @common
+Scenario: Content copying can be cancelled
+  Given I navigate to content "Test Article edited4" in "Media/Images"
+  When I click on the edit action bar button "Copy"
+    And I open UDW and go to "Home"
+    And I close the UDW window
+  Then I should be on content page "Test Article edited4" in "Media/Images"
+    And Content attributes equals
+      | attribute | value                      |
+      | Title     | Test Article edited4       |
+      | Intro     | Test article intro edited4 |
+    And there's no "Test Article edited2" "Article" on "Sub-items" list
+
+@javascript @common
+Scenario: Content can be copied
+  Given I navigate to content "Test Article edited4" in "Media/Images"
+  When I click on the edit action bar button "Copy"
+    And I open UDW and go to "Home"
+    And I confirm the selection in UDW
+  Then I should be on content page "Test Article edited4" in "Home"
+    And Content attributes equals
+      | attribute | value                      |
+      | Title     | Test Article edited4       |
+      | Intro     | Test article intro edited4 |
+
+@javascript @common
+Scenario: Content can be moved to trash
+  Given I navigate to content "Test Article edited4" in "Media/Images"
+  When I click on the edit action bar button "Send to Trash"
+    And I confirm sending to trash
+    And I go to "Test Article edited4" in "Home"
+    And I click on the edit action bar button "Send to Trash"
+    And I confirm sending to trash
+    And I click on the side menu bar button "Trash"
```

Comment:
I'd check here that the item is visible in Trash and end the Scenario - leaving Trash actions to a separate story. Adding a check that `Test Article edited4` is not visible in Home sub-items would be nice too (same goes for a notification, if there is one).

If the trashed item were never shown in Trash (the list stays empty) this test would still pass.